css-four-column-layout/styles.css

```diff
@@ -0,0 +1,41 @@
+.container{
+ background-color: rgb(218, 218, 218);
+ margin: 0 1rem;
+
+}
+
+.heading{
+  text-align: center;
+  font-family: 'Barlow', sans-serif;
+  font-weight: bolder;
+}
+.row{
+  display: flex;
+  font-family: 'Barlow', sans-serif;
+}
+
+.column{
+  width: 20%;
+  margin: 0 1.5rem;
+  justify-content: space-around;
+}
+
+.phone-img{
+  width: 12rem;
+}
+
+.icon-img{
+  width: 3rem;
+  margin-left: 4.5rem;
```

Comment:
😱 😱 😱 😱 😱 

Is this being used to center the icon? 

_**Infomercial Voice:**_ There HAS to be a better way! 😩